coops/titans_stronghold.tex

```diff
@@ -0,0 +1,199 @@
+\textit{A long time ago, a powerful fortress was built to house the fantastic and murderous Titans.
+  No one has ever entered and returned to tell the tale.
+  Now, people speak about the death of guardians and an incredible amount of treasures, potentially including the best bow in the world.\\
+  If this legend of the empty fort is true, it's time for excavation.\\
+  But for now, you and your friends need to find the keys spread around Antagarich to open the gate of the Titans' stronghold.
+}
+\subsection*{\MakeUppercase{Scenario length}}
+
+This scenario plays out over 16 rounds (15 in Impossible difficulty).
+
+\subsection*{\MakeUppercase{Player setup}}
+
+\textbf{Player Count:} 1 - 6
+
+\textbf{Starting Resources:} 30 \svg{gold}, 8 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Units:} 
+\begin{itemize}
+  \item A Pack of \svg{bronze} units of your choice.
+  \item A Few \svg{bronze} units of your choice.
+\end{itemize}
+
+\textbf{Town Buildings:} None
+
+\subsection*{\MakeUppercase{Map setup}}
+
+Take the following Map Tiles and set them up as shown in the scenario map layout (P for the number of players):
+
+\textbf{(1*P) Starting Map tile (I)}
+\begin{itemize}
+    \item Starting Tiles of your chosen factions.
+    \item Ignore all of their yellow borders.
+\end{itemize}
+
+\textbf{(2*P) × Far Map tile (II-III)}
+
+\textbf{(2*P) × Near Map tile (IV-V)}
+
+\textbf{(1*P) + 1 × Center Map tile (VI-VII)}
+
+If you don't have enough VI-VII tiles, you can use another tile near to tile I.
+The center of this tile is the Titans' Stronghold.
+
+\subsection*{\MakeUppercase{Win Conditions}}
+
+\textbf{Win:} Kill all units in the Titans' Stronghold (the center of tile VI-VII next to the tiles I).
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{4$^{th}$, 8$^{th}$ and 12$^{th}$ Rounds:}
+\begin{itemize}
+    \item Remove black cubes from Windmills, Water Wheels, and Mystical Gardens.
+\end{itemize}
+
+\end{multicols*}
+
+\newpage
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\begin{itemize}
+    \item Remember: the center of tile VI-VII next to all I tiles is the Titans' Stronghold.
+    \item Nobody can enter the Titans' Stronghold before all other VII fields are flagged by any player.
+    \item After defeating a level VII neutral army, instead of resolving the field, the player chooses an option three times from the following list (an option may be chosen multiple times):
+    \begin{itemize}
+        \item Another player (you choose) gains 5 gold.
+        \item Another player (you choose) gains 2 building materials.
+        \item Another player (you choose) gains 1 valuable.
+    \end{itemize}
+    Then flag the VII field with a faction cube.
+    (In solo play: no bonus!)
+    \item Ignore all yellow borders in tiles I.
+    \item You can use your build token to give your resources to another player.
```

Comment:
How exactly does that work? You flip the build token and give any number of resources to any player?

Reply:
Yep, and with flip build token you can give to two or more player